relay-event-schema/src/protocol/breadcrumb.rs

```diff
@@ -116,6 +116,14 @@ pub struct Breadcrumb {
     #[metastructure(skip_serialization = "null")]
     pub event_id: Annotated<EventId>,
 
+    /// The origin of the breadcrumb. _Optional_, defaults to `null`.
+    ///
+    /// A string representing the origin of the breadcrumb. This is typically used to identify the
+    /// source of the breadcrumb. For example hybrid SDKs can identify native breadcrumbs from
+    /// JS or Flutter.
+    #[metastructure(skip_serialization = "null")]
```

Comment:
```suggestion
    #[metastructure(skip_serialization = "empty")]
```

Strings are usually skipped when empty.